packages/cli/src/commands/init/template.js

```diff
@@ -4,10 +4,7 @@ import path from 'path';
 import * as PackageManager from '../../tools/packageManager';
 import {logger} from '@react-native-community/cli-tools';
 import copyFiles from '../../tools/copyFiles';
-
-const RegexEscape = function(regex) {
-  return regex.replace(/[-\/\\^$*+?.()|[\]{}]/g, '\\$&');
-};
+import {replacePathSepForRegex} from 'jest-regex-util';
```

Comment:
It's in our development deps. Jest libs will usually be there but there's not guarantee. If we import a package, we need to declare it as our dependency. So, either copy the oneliner from jest-regex-util or add it to the deps of packages/cli/package.json. I'd prefer to copy to avoid users having multiple versions of jest packages :)

Reply:
Oh okay, got it. I will make a copy.